debezium-server-kinesis/src/main/java/io/debezium/server/kinesis/KinesisChangeConsumer.java

```diff
@@ -106,22 +112,94 @@ void close() {
     @Override
     public void handleBatch(List<ChangeEvent<Object, Object>> records, RecordCommitter<ChangeEvent<Object, Object>> committer)
             throws InterruptedException {
-        for (ChangeEvent<Object, Object> record : records) {
-            LOGGER.trace("Received event '{}'", record);
 
+        // Split the records into batches of size 500
+        String streamName = records.get(0).destination();
```

Comment:
Add guard just in case the `records` is empty list. Should not happen but better to apply it as defensive policy.
Also what would happen if there will be records for multiple tables/destinations in the list?